CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [Developer]: Fixed deleting a project with project subscriptions. See [PR 1348](https://github.com/phac-nml/irida/pull/1348)
 * [Developer]: Updated OAuth2 implementation to use Spring Security 5 OAuth2 libraries. See [PR 1339](https://github.com/phac-nml/irida/pull/1339)
 * [Developer]: Add identifier to project drop-downs. See [PR 1352](https://github.com/phac-nml/irida/pull/1352)
+* [ALL]: Added LDAP/ADLDAP support.
```

Comment:
Is this changelog comment going to be updated to include PR reference in the main branch (i.e. ldap)

Reply:
Yes that is the plan. Once we have all the features in the `ldap` branch and it's ready for a PR to dev it will be added.